.github/workflows/ci.yml

```diff
@@ -73,31 +73,24 @@ jobs:
           working-directory: backend
           files: ./coverage.xml
 
-#  frontend-build:
-#    name: Create 'production' build
-#    runs-on: ubuntu-latest
-#
-#    defaults:
-#      run:
-#        working-directory: frontend
-#
-#    steps:
-#      - uses: actions/checkout@v4
-#      - uses: actions/setup-node@v3
-#        with:
-#          node-version-file: 'frontend/.nvmrc'
-#
-#      - name: Build Javascript
-#        run: |
-#          npm ci --legacy-peer-deps
-#          npm run build
-#
-#      - name: Store build artifact
-#        uses: actions/upload-artifact@v3
-#        with:
-#          name: open-archiefbeheer-frontend-build
-#          path: dist/
-#          retention-days: 1
+  frontend-build:
+    name: Create 'production' build
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        working-directory: frontend
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version-file: 'frontend/.nvmrc'
+
+      - name: Build Javascript
+        run: |
+          npm ci --legacy-peer-deps
+          npm run build
```

Comment:
Don't we want to start storybook and run the stories as tests?

Reply:
Yes! That is definitely also a good test to add. But I wanted to make sure that `npm run build` runs with no errors, otherwise it will fail in the step of creating the docker image.